### ISSUE TITLE
Download records page

### ISSUE THREAD

**Issue by TestPolygon** [leoding86/webextension-pixiv-toolkit#79]
**I think that it's useless without images (the previews).**

And I think it should looks like the old version of the history page. The pagination is much better for the big list of entries. The infinity scroll is not convenient. I don't like "the new style" of the history page. Just fix this bug: https://github.com/leoding86/webextension-pixiv-toolkit/issues/75.

I have already posted [here](https://github.com/leoding86/webextension-pixiv-toolkit/issues/75#issuecomment-667672258) the message about it.

**TL;DR**
I need the old styled history page*, but not for every watched art, but only for downloaded ones.

_*The page with the pagination (it's very important for big lists) and with the more compact visible entries._

**Comment by leoding86** [comment]
No plan to change the layout.
